apps/meteor/tests/end-to-end/api/livechat/contacts.ts

```diff
@@ -632,4 +632,54 @@ describe('LIVECHAT - contacts', () => {
 			expect(res.body.errorType).to.be.equal('invalid-params');
 		});
 	});
+
+	describe('Contact Channels', () => {
+		let visitor: ILivechatVisitor;
+
+		beforeEach(async () => {
+			visitor = await createVisitor();
```

Comment:
You're creating a visitor here on the `beforeEach` but also creating inside of each test directly;
Due to that, only one of the visitors created gets deleted in the afterEach.